tests/snapshots/solver__unsat_constrains_2.snap

```diff
@@ -1,21 +1,10 @@
 ---
 source: tests/solver.rs
-assertion_line: 703
 expression: error
 ---
 The following packages are incompatible
 |-- a * cannot be installed because there are no viable options:
-    |-- a 2 would require
-        |-- b *, which cannot be installed because there are no viable options:
-            |-- b 2 would require
-                |-- c >=2, <3, which cannot be installed because there are no viable options:
-                    |-- c 2 would constrain
-                        |-- a >=3, <4 , which conflicts with any installable versions previously reported
-            |-- b 1 would require
-                |-- c >=1, <2, which cannot be installed because there are no viable options:
-                    |-- c 1 would constrain
-                        |-- a >=3, <4 , which conflicts with any installable versions previously reported
-    |-- a 1 would require
+    |-- a 1 | 2 would require
```

Comment:
Some information is lost here. Why cant 'b 1' not be used for the 'b *' requirement?

Reply:
The diff is misleading. If you look at the whole file you should see information about `b 1`, I think.

Reply:
Indeed! Looks great!